### ISSUE TITLE
[Request] Minetweaker/Modtweaker support for CC recipe handlers

### ISSUE THREAD

**Issue by Shahelion** [ReikaKalseki/Reika_Mods_Issues#1226]
Only creating recipes, not removing existing ones, of course.

Including alloying, boosting, and the various tiers of casting.

I love the whole crafting system, and it's a shame that I can't make my own recipes for other things with it.

**Comment by ReikaKalseki** [comment]
This is a desired feature but is contingent on getting the RC handlers working first.

**Comment by AshIndigo** [comment]
I cant find this any but I swear you added a custom recipe for autonomous activators with multiblock casting

**Comment by Shahelion** [comment]
Reika adding recipes using his handlers is different from anyone else being able to add recipes using his handlers.

**Comment by redelman** [comment]
That custom autonomous activator recipe was also on Reika's dev server only.

**Comment by Shahelion** [comment]
I didn't close this...wat.

**Comment by ReikaKalseki** [comment]
No longer needed.

**Comment by Shahelion** [comment]
Does that mean it's possible to do now? Or via some other third mod?

**Comment by ReikaKalseki** [comment]
https://www.reddit.com/r/feedthebeast/comments/5o0a2m/by_request_custom_recipe_configs/

**Comment by Builderb0y** [comment]
Is this recipe system reloadable from in-game?

**Comment by ReikaKalseki** [comment]
No. RC recipe systems do not support that.

**Comment by Builderb0y** [comment]
Then I think there still is something to gain from having official minetweaker support. Changing configs over and over again until you get it right is a pain, and testing to make sure that they actually work after being changed is equally so. Both of these problems are MUCH easier to work with using minetweaker, since you can see your results immediately.

**Comment by OvermindDL1** [comment]
Cool.  :-)
I still need to tweak my family server as well, roterycraft/chromaticraft/etc... stuff is *way* too easy to get for the hard-mode setup I have...

**Comment by ReikaKalseki** [comment]
> Then I think there still is something to gain from having official minetweaker support.

MT support would add nothing. *The fundamental recipe systems in RC are not dynamic. Changes **cannot** take effect after game load.*
